package.json

```diff
@@ -36,6 +36,7 @@
     "chromedriver": "91.0.1",
     "commitizen": "4.2.4",
     "css-loader": "3.6.0",
+    "cypress": "^7.3.0",
```

Comment:
Might as well use Cypress 8, unless there are some breaking changes.

Reply:
did not know that was rekeased